.circleci/config.yml

```diff
@@ -117,17 +124,37 @@ build_images: &build_images
     - checkout
     - <<: *exports
     - run: |
+        mkdir -p images/
         read -ra IMG_ARRAY <<< "$IMAGES"
         if [[ -n "${CIRCLE_TAG}" ]]; then
           makeArgs="VERSION=${CIRCLE_TAG}"
         fi
         for IMAGE in "${IMG_ARRAY[@]}"; do
-          make IMG_MODIFIER="$IMG_MODIFIER" IMAGE_TAG="${DEV_TAG}" $makeArgs ${IMAGE}
+          make IMG_MODIFIER="$IMG_MODIFIER" IMAGE_TAG="${DEV_TAG}" $makeArgs kubeapps/${IMAGE}
           if [[ -z "${CIRCLE_PULL_REQUEST}" && -n "${DOCKER_USERNAME}" && -n "${DOCKER_PASSWORD}" ]]; then
             docker login -u="${DOCKER_USERNAME}" -p="${DOCKER_PASSWORD}"
-            docker push ${IMAGE}${IMG_MODIFIER}:${DEV_TAG}
+            docker push kubeapps/${IMAGE}${IMG_MODIFIER}:${DEV_TAG}
           fi
+          docker save kubeapps/${IMAGE}${IMG_MODIFIER}:${DEV_TAG} > images/${IMAGE}${IMG_MODIFIER}:${DEV_TAG}
```

Comment:
We only need to do this if this is a pull request, right?

Reply:
It's mandatory when it is a PR. When running this in master we could pull the images but this is convenient because loading the images from the local file is faster.

Reply:
Ah I was thinking we only do the local test for PRs, but we actually run it all the time. sg.